.eslintrc.json

```diff
@@ -38,7 +38,8 @@
     "@typescript-eslint/no-non-null-assertion": "error",
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
-    "react/prop-types": "off"
+    "react/prop-types": "off",
+    "prettier/prettier": ["error", {"endOfLine": "auto"}]
```

Comment:
What problem did this solve for you? Also I think if we change this it should be done in `.prettierrc.json`, but could we do that in a different PR?

Reply:
This gives a green slate when doing a git clone on Windows where the line endings are different.

Reply:
Removed

Reply:
This would be better managed with git rather than prettier IMO

Reply:
Yeah I'd forgotten how we do this with cucumber-js and now recall that it's https://github.com/cucumber/cucumber-js/blob/main/.gitattributes#L1